index.html

```diff
@@ -0,0 +1,23 @@
+
+<!-- About ME, John Gutierrez -->
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Resume</title>
+</head>
+<body>
+  <!-- Your content goes here -->
+  <h1>Hi, my name is John Gutierrez</h1>
+  <p>I am pursuing a degree in computer science. I'm very involved in a few industries and still learning about Game development, Web Development, and Software Engineering.</p>
+  <ul>
+    <li>Games</li>
+    <li>Web Projects</li>
+  </ul>
+  <a href="https://github.com/">This is my GitHub</a>
+  <a href="https://www.instagram.com/">This is my Instagram</a>
+  <a href="https://www.example.com/">This is my portfolio</a>
+  <p> email me at  <a href = "mailto:carlos.will51@gmail.com">carlos.will51@gmail.com</a></p>
```

Comment:
as a personal opinion make sure to add your Linkedin profile 👍🏽

Reply:
will do next time, i think my next commit is out? not sure honestly still getting the hang of this